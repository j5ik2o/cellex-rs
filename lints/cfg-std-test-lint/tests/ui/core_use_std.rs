#![deny(cfg_std_test)]

use std::time::Duration;

fn main() {
  let _ = Duration::from_secs(1);
}
