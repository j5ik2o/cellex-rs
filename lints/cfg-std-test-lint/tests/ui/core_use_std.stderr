error: `use std::...` の使用が検出されました
  --> $DIR/core_use_std.rs:3:1
   |
LL | use std::time::Duration;
   | ^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: std 名前空間のアイテムは std 対応クレートへ移動するか、該当箇所で `#![allow(cfg_std_test)]` を付与してください
   = note: AI向けアドバイス: core/embedded コードでは `std` 依存を避け、必要に応じて `alloc` など代替を検討しましょう。
note: the lint level is defined here
  --> $DIR/core_use_std.rs:1:9
   |
LL | #![deny(cfg_std_test)]
   |         ^^^^^^^^^^^^

error: aborting due to 1 previous error

